src/components/Videos.vue

```diff
@@ -129,6 +139,20 @@ export default {
 	},
 
 	methods: {
+		hideHeaderAndFooter(e){
+			// work arround to get the state of the fullscreen button, aria-selected attribute is not reliable
+			this.fullscreenButton = !this.fullscreenButton
+			if(this.fullscreenButton) {
+				document.body.querySelector('footer').style.display = 'none'
+				document.body.querySelector('main').style.height = '100vh'
+				document.body.querySelector('main').style.width = '100vw'
+				document.body.querySelector('main').style.margin = '0'
+			}
+			else {
+				document.body.querySelector('footer').removeAttribute('style')
+				document.body.querySelector('main').removeAttribute('style')
```

Comment:
won't we loose all other styles which were set on `main` and `footer`?

Reply:
No we don't have any styles set inline